.github/workflows/visual-regression.yml

```diff
@@ -0,0 +1,58 @@
+name: Visual regression with Netlify
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
- Visual regression test to help review visualization features? If so, I think it may not need to be triggered by master push. 
- Also, I was wondering if it would be ok to only start it when there is a change in the visualization module.
- Furthermore, I think there are cases where we may want to start it manually, so I think it would be better to add `workflow_dispatch`.

Reply:
Sounds reasonable. In additioin, we can save the usage of netlify by restricting the trigger targets.
I agree with @HideakiImamura.